src/main/java/pl/betoncraft/betonquest/config/ConfigUpdater.java

```diff
@@ -130,21 +130,21 @@ public ConfigUpdater() {
         // if the version is null the plugin is updated from pre-1.4 version
         // (which can be 1.0, 1.1 or 1.2)
         if (version == null || version.equals("1.4")) {
-            updateTo1_4_1();
+            updateTo1dot4dot1();
         } else if (version.equals("1.4.1")) {
-            updateTo1_4_2();
+            updateTo1dot4dot2();
         } else if (version.equals("1.4.2")) {
-            updateTo1_4_3();
+            updateTo1dot4dot3();
         } else if (version.equals("1.4.3")) {
-            updateTo1_5();
+            updateTo1dot5();
         } else if (version.equals("1.5")) {
-            updateTo1_5_1();
+            updateTo1dot5dot1();
         } else if (version.equals("1.5.1")) {
-            updateTo1_5_2();
+            updateTo1dot5dot2();
         } else if (version.equals("1.5.2")) {
-            updateTo1_5_3();
+            updateTo1dot5dot3();
         } else if (version.equals("1.5.3") || version.equals("1.5.4") || version.equals("1.6")) {
-            updateTo1_6();
+            updateTo1dot6();
```

Comment:
The old names are much more readable and `_` wasn't used to separate words but instead replace special characters. I believe that this violation has to be suppressed instead of "fixed".

Reply:
The Updater will be dropped anyways, no need to spend any more time on it.